pkg/operator/controller/prometheus.go

```diff
@@ -192,9 +196,10 @@ func getAlertRules() []promv1.Rule {
 				"runbook_url": runbookURLBasePath + "CDIStorageProfilesIncomplete",
 			},
 			map[string]string{
-				severityAlertLabelKey:  "info",
-				partOfAlertLabelKey:    partOfAlertLabelValue,
-				componentAlertLabelKey: componentAlertLabelValue,
+				severityAlertLabelKey:     "info",
+				healthImpactAlertLabelKey: "warning",
```

Comment:
@assafad  We should consider this as even a critical impact on operator health, since the  CDI cannot create a VM disk on the PVC. Its an important functionality. But I think someone from the CDI team should advise here.

Reply:
I can see arguments for both, but I am in favor of `warning` since it's still possible to specify the missing properties manually on the DV object